tests/untl_structure_test.py

```diff
@@ -511,15 +503,16 @@ def test_Metadata_create_xml_file(tmpdir):
     xml_file = tmpdir.join('untl.xml')
     metadata.create_xml_file(xml_file.strpath)
     assert xml_file.read() == (
-        u'<?xml version="1.0" encoding="UTF-8"?>\n'
-        u'<metadata>\n'
-        u'  <title qualifier="seriestitle">Colecci&#243;n</title>\n'
-        u'  <contributor qualifier="cmp">\n'
-        u'    <type>per</type>\n'
-        u'    <name>Oudrid, C. (Crist&#243;bal), 1825-1877.</name>\n'
-        u'  </contributor>\n'
-        u'  <description qualifier="content">"Fortuna te d&#233; Dios, hijo"</description>\n'
-        u'</metadata>\n'
+        """<?xml version='1.0' encoding='utf-8'?>
```

Comment:
While I don't think changing from concatenated strings to a triple-quoted strings _helps_ the readability of this XML chunk, I don't think it hurts it that much either. However, if you do want to make this switch, then the surrounding parentheses no longer serve any purpose.

Reply:
I changed it 'cos I used triple-quoted strings in `test_Metadata_create_xml_string` test function for asserting the expected_text. I can get rid of parentheses.

Reply:
Well, it turns out my comment is actually wrong :smile: , we _do_ need the parentheses here since the first line is not on the same line as the assignment. However, given the reasoning, I think we should revert this to what it was. It's ok to instantiate different strings using different ways, since some strings have indentation across lines (like this one) and some don't. I think priority here should be given to whichever way best shows the intended content of the string, and I think the way it was did pretty well. How do you feel about reverting it to the previous way, minus all the `u` prefixes?

Reply:
Well...I've observed that the double quotes for xml version and encoding are giving assertion errors, I tried escaping them too but that didn't help either. It happened for other assertions too!

Reply:
Even `"""<?xml version=\"1.0\" encoding=\"UTF-8\"?>`?

Reply:
Yes even that's giving me assertion Error!

Reply:
heh, I think we're commenting on the wrong thread :smile: